pom.xml

```diff
@@ -45,7 +45,7 @@
         <air.java.version>1.8.0-40</air.java.version>
 
         <dep.antlr.version>4.5.1</dep.antlr.version>
-        <dep.airlift.version>0.125</dep.airlift.version>
+        <dep.airlift.version>0.126-SNAPSHOT</dep.airlift.version>
```

Comment:
This depends on https://github.com/airlift/airlift/pull/396 

If this seems reasonable, we can release airlift and update this dependency to 126